modules/ROOT/pages/security/ip-filter.adoc

```diff
@@ -0,0 +1,55 @@
+[[aura-reference-security]]
+= IP Filter
+
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+IP Filters restrict access to your Aura instance over the public internet by allowing only trusted IP addresses. 
+This feature helps secure your environment without requiring private network configurations.
+
+* Aura Business Critical supports up to 20 allowed IP Ranges.
+* Aura Virtual Dedicated Cloud supports up to 100 allowed IP Ranges.
+* Higher capacity options are available on request.
+
+Contact support if you need to increase the default IP range limits.
+
+== Required roles
+
+Users with the xref:/aura/user-management.adoc[roles] `Organization Owner` or `Organization Admin` can create and edit IP Filters from the xref:visual-tour/#org-settings.adoc[organization settings] page.
+
+== Add an IP Filter
+
+. In the Aura console, go to *Organization > Settings > Security & Networking > IP Filtering*
+. *Select entities* - Choose where to apply the filter:
+.. *Organization* – applies to all instances in the org
+.. *Project* – applies to all instances in the project
+.. *Instance* – applies to individual instances
+. Select allowed IP addresses:
+.. *All* – No filtering is applied, all IPs are allowed.
+.. *Specific range of IP addresses* – Enter individual IP addresses or CIDR ranges.
+
+=== How filters are applied
+
+IP Filters allow or deny a connection to an instance. 
+An instance can have one IP Filter only.
+If a filter is already applied, the instance’s checkbox will be greyed out.
+
+Applying filters at the organization or project level helps admins set filters for multiple instances and newly created instances.
+When applied at the organization or project level, new entities created within that scope will automatically inherit the filter settings.
+
+== Edit an IP Filter
+
+To edit an existing filter configuration, use the *Inspect* button.
+
+== Use cases 
```

Comment:
Maybe try to rewrite this section to highlight the pros and then add a link to Private endpoints?